ml/airflow/.env.example

```
@@ -0,0 +1,20 @@
+AIRFLOW__CORE__EXECUTOR=LocalExecutor
+AIRFLOW_DB_USER=airflow
+AIRFLOW_DB_NAME=airflow
+AIRFLOW__DATABASE__SQL_ALCHEMY_CONN=postgresql+psycopg2://${AIRFLOW_DB_USER}:${AIRFLOW_DB_PASSWORD}@postgres:5432/${AIRFLOW_DB_NAME}
+AIRFLOW__DATABASE__SQL_ALCHEMY_CONN_ASYNC=postgresql+asyncpg://${AIRFLOW_DB_USER}:${AIRFLOW_DB_PASSWORD}@postgres:5432/${AIRFLOW_DB_NAME}
+AIRFLOW__CORE__FERNET_KEY=<generate-a-valid-fernet-key>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AIRFLOW__CORE__FERNET_KEY key should go before the AIRFLOW__DATABASE__SQL_ALCHEMY_CONN_ASYNC key

(UnorderedKey)